sdk/nodejs/src/eventQueue.ts

```diff
@@ -121,6 +124,8 @@ export class EventQueue {
 
     cleanup(): void {
         clearInterval(this.flushInterval)
+        this.bucketing.flushEventQueue(this.sdkKey)
```

Comment:
what's the idea behind calling this here?

Reply:
basically to flush out anything in the bucketing queue so it doesn't stay there, it doesn't actually make a request it just doesn't use the return data in anything, it just throws it away

Reply:
might be worth adding a comment about that

Reply:
also I dont think this will actually have any effect on memory since this just moves the events to a different internal queue inside the bucketing lib

Reply:
hmm yeah, I've thought about closing the event queue but that just causes problems when calling variables since it tries to get the event queue when evaluating a variable